### PR TITLE
Feature/flowerbed repository tests

### LINKED ISSUE #204: Implement flowerbed repository tests

### PR DESCRIPTION
**Pull request by doriengr**
Close #204

### CODE REVIEW

File: internal/storage/postgres/flowerbed/get_test.go [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+
+type testFlowerbeds struct {
+	ID          int32
+	Description string
+	RegionID    int32
+	SensorID    int32
+	Images      []*entities.Image
+}
+
+var allTestFlowerbeds = []*testFlowerbeds{
```
Comment (Zoka-tech) [inline internal/storage/postgres/flowerbed/get_test.go:415]: Maybe seperate the "fake" Flowerbed Data into its own file to have this a bit more organized?

**Review by choffmann — CHANGES_REQUESTED** [review]
:fire:

File: internal/storage/postgres/flowerbed/flowerbed.go [hunk 2]
```diff
@@ -149,13 +160,36 @@ func (r *FlowerbedRepository) UnlinkImage(ctx context.Context, id, imageID int32
...
+		return err
+	}
+
+	return r.checkFlowerbedExists(rowID)
+}
+
+func (r *FlowerbedRepository) checkFlowerbedExists(rowID int32) error {
+	if rowID == 0 {
+		return storage.ErrFlowerbedNotFound
+	}
```
[the remark this answers is not in the document]
Comment (doriengr) [reply internal/storage/postgres/flowerbed/flowerbed.go:193]: Perhaps the implementation is not correct, but the reason for this check are the tests in which a non-existent id is overgiven and it should return an error. Before the implementation the return value was always nil.  

I based the structure on the Delete tree cluster.
https://github.com/green-ecolution/green-ecolution-backend/blob/e5cd1d9a2b3a62e75eb4bdff98ae9d66217b0f5f/internal/storage/postgres/treecluster/treecluster.go#L125

[the remark this answers is not in the document]
Comment (doriengr) [reply internal/storage/postgres/flowerbed/flowerbed.go:193]: What is the correct way?

**Review by choffmann — APPROVED** [review]
:+1:

**Pull request merged**